packages/@react-stately/list/src/useListState.ts

```diff
@@ -47,7 +47,7 @@ export function useListState<T extends object>(props: ListProps<T>): ListState<T
 
   let factory = nodes => nodeFilter ? new ListCollection(nodeFilter(nodes)) : new ListCollection(nodes as Iterable<Node<T>>);
 
-  let collection = useCollection(props, factory, null, [nodeFilter]);
+  let collection = useCollection(props, factory, [], nodeFilter);
```

Comment:
Not entirely sure why a few of combobox tests failed without adding this empty array as a context value...